### PR TITLE
[KeyVault] Add functionality to import a key from string

### PR DESCRIPTION
**Pull request by khaliddermoumi**
**Description**  
This PR implements the feature request #13685 

**Testing Guide**  
Just run the test suite, f. e. "azdev test keyvault"

**History Notes**  
[Keyvault] 'az keyvault key import' now supports importing from strings via two new parameters.

### CODE REVIEW

**Comment by yonzhan** [comment]
KeyVault

**Comment by bim-msft** [comment]
Hi @khaliddermoumi , thanks for your contribution!
However, there are some linter issues in your code, you can fix them and update the code to run CI again:
image

**Comment by khaliddermoumi** [comment]
Hi @bim-msft ,
the build is green now.

File: src/azure-cli/azure/cli/command_modules/keyvault/_validators.py [hunk 1]
```diff
@@ -127,14 +127,19 @@ def validator(ns):
...
-    if (not byok_file and not pem_file) or (byok_file and pem_file):
-        raise ValueError('supply exactly one: --byok-file, --pem-file')
-    if byok_file and pem_password:
-        raise ValueError('--byok-file cannot be used with --pem-password')
-    if pem_password and not pem_file:
-        raise ValueError('--pem-password must be used with --pem-file')
+    if not ((byok_file and not byok_string and not pem_file and not pem_string) or
+            (not byok_file and byok_string and not pem_file and not pem_string) or
+            (not byok_file and not byok_string and pem_file and not pem_string) or
+            (not byok_file and not byok_string and not pem_file and pem_string)):
```
Comment (bim-msft) [inline src/azure-cli/azure/cli/command_modules/keyvault/_validators.py:137]: As we have 4 args now, it's not a good way to list all the conditions. This is the recommended implementation:
```python
if len([arg for arg in [byok_file, byok_string, pem_file, pem_string] if arg]) != 1:
```

**Comment by khaliddermoumi** [comment]
Thx for accepting the pull request! :)

**Pull request merged**